database/azure_blob_storage/save_get_blob.py

```diff
@@ -313,30 +313,32 @@ def save_news_index_to_db(blob_name: str, json_data,  session) -> None:
         for team_name in teams:
             if isinstance(team_name, list):
                 for name in team_name:
-                    team_index_id = team_dict.get(name, None)
-                    if team_index_id is not None:
-                        team_index = TeamInNews(
-                            news_id=news_index.news_id,
-                            name=name,
-                            team_index_id=team_index_id
-                        )
-                        session.add(team_index)
+                    __add_team_index_to_db(team_dict, name, news_index, blob_name, session)
```

Comment:
We have two identical lines.

You can do smth like:

`if not isinstance...:`
`    team_name = [team_name,]`
` `
`for name in team_name:`
`    your logic`